tests/fast-integration/skr-kyma-to-kyma2-upgrade/upgrade-helpers.js

```diff
@@ -72,7 +72,7 @@ async function ensureOrchestrationSucceeded(kcpconfigPath, orchenstrationID) {
       () => getOrchestrationStatus(kcpconfigPath, orchenstrationID),
       (res) => res && res.state && (res.state === "succeeded" || res.state === "failed"),
       1000*60*15, // 15 min
-      1000 * 5 // 5 seconds
+      1000 * 30 // 5 seconds
```

Comment:
nit: I think the comment should now also read `30 seconds` :)

Reply:
hmm, I guess I should always put `/hold`? There were some failed PR gates so I felt comfortable giving approval thinking it won't merge prematurely but it merged despite the checks being red...